modules/pubmaticBidAdapter.js

```diff
@@ -710,6 +618,24 @@ function _handleEids(payload) {
     payload.user.eids = eids;
   }
 }
+// TODO: need a better way to identify mediaType from response.
+function _checkMediaType(adm, newBid) {
+  // Create a regex here to check the strings
```

Comment:
todo: use regex for vast

Reply:
done

Reply:
Hello @pm-manasi-moghe ,
Let's keep this comment unresolved until we merge the changes into this branch.